canary/producer-cpp/jobs/orchestrator.groovy

```diff
@@ -0,0 +1,217 @@
+import jenkins.model.*
+
+WORKSPACE_PRODUCER="src"
+// WORKSPACE_CONSUMER="consumer-java"
+GIT_URL='https://github.com/aws-samples/amazon-kinesis-video-streams-demos.git'
+GIT_HASH='new-canary-producer-cpp'
+RUNNER_JOB_NAME_PREFIX = "producer-cpp-runner"
+
+// TODO: Set up configurability to run different parameter combinations
+// Run long run canary for 12 hours
+LONG_RUN_DURATION_IN_SECONDS = 12 * 60 * 60
+SHORT_RUN_DURATION_IN_SECONDS = 30
+COLD_STARTUP_DELAY_IN_SECONDS = 60 * 60
+MIN_RETRY_DELAY_IN_SECONDS = 60
+FRAGMENT_SIZE_IN_BYTES = 250000
```

Comment:
Is this needed for the canary?

Reply:
Not needed, but it seemed like a useful thing to keep from the C Canary, it makes it easy to set the typical durations, delays, etc. I set the env vars for Canary duration this using these definitions